### DISCUSSION TITLE
Request: Add option to look up "spy"

### DISCUSSION THREAD

**Discussion by CypherPhyre** [Zrips/CMI#315] [Feature Requests]
It was recently brought to my attention that one of my staff misunderstood how to use the commandspy command and invertedly enabled it on another player by adding the player's name to the command. It wasn't until the player started asking what the "spy thing" was that we knew something was wrong. I would like you to add a way to look up if the "spy" commands are enabled on players. Only staff have perms for the commandspy and socialspy. I have not been able to find a way to check that in the config or commands. It was only when a player started asking about it that we became aware. The staff member WAS Opped at the time if that plays any part.

**Comment by Zrips** [comment]
Duno if we really need separate command for this while you could simply look up at your data base and at Spy column and sort it. Anyone with 1 in it, has social spy enabled.

**Comment by CypherPhyre** [comment]
I ended up doing that with the help of a co-owner. I was hoping for an easier way to quickly check without the need to comb through a lot of database entries. It doesn't have to be limited to the spy commands. You could make it like the /cmi haspermission and do /cmi hasenabled

**Comment by LogGits** [comment]
A seperate command isn't needed. Nonetheless, a placeholder can be added into the /cmi info page. 

Something like "Spying: (true/false)"
